src/undate/undate.py

```diff
@@ -123,10 +124,11 @@ def calculate_earliest_latest(self, year, month, day):
         if month == "XX":
             month = None
 
-        min_month = 1  # is min month ever anything other than 1 ?
-        # get max month from the calendar, since it depends on the
-        # calendar and potentially the year (e.g. leap years in Hebrew Anno Mundi)
-        max_month = self.calendar_converter.max_month(max_year)
+        # get first and last month from the calendar, since it is not
+        # always 1 and 12
+        # TODO need to differentiate between min/max and first/last!
```

Comment:
Is this also an issue in Hebrew? It looks like there are two different start months (Tishri which is ~September for calendar year, and Nisan for religious year which is ~March/April?)

Reply:
I think it only applies to Hebrew calendar and not Hijri - but maybe others I'm not aware of